compiler/plugins/input/Torch/InputConversion/test/torch_to_iree.mlir

```diff
@@ -29,7 +29,7 @@ util.global private @_params.classifier.bias {inlining_policy = #util.inline.nev
 // -----
 
 // Verify we can decompose complex ops
-// CHECK-LABEL: util.func public @main$async(%arg0: !hal.buffer_view, %arg1: !hal.fence, %arg2: !hal.fence) -> (!hal.buffer_view, !hal.buffer_view)
+// CHECK-LABEL: util.func public @main(%arg0: !hal.buffer_view) -> (!hal.buffer_view, !hal.buffer_view)
```

Comment:
Could we have a "CHECK-NOT" here for hal.fence and hal.barrier? Just to make sure we are not still accidentally creating those? They should be gone right?